torch/testing/_internal/common_methods_invocations.py

```diff
@@ -7346,9 +7356,9 @@ def sample_inputs_diagflat(op_info, device, dtype, requires_grad, **kwargs):
 
 def sample_inputs_max_unpool(op_info, device, dtype, requires_grad, **kwargs):
     unpool_name_to_pool_method_dict = {
-        'nn.functional.max_unpool1d': torch.nn.functional.max_pool1d,
-        'nn.functional.max_unpool2d': torch.nn.functional.max_pool2d,
-        'nn.functional.max_unpool3d': torch.nn.functional.max_pool3d
+        'nn.functional.max_unpool1d': torch.nn.functional.max_unpool1d,
```

Comment:
This looks... wrong? It's an unpool to pool mapping no?

Reply:
yeah